os-rabbitmq.te

```diff
@@ -32,3 +32,6 @@ allow logrotate_t self:passwd passwd;
 allow logrotate_t proc_net_t:file read;
 allow logrotate_t self:process setrlimit;
 allow rabbitmq_t proc_net_t:file read;
+
+# Bugzilla 2056565
+allow rabbitmq_t tmpfs_t:file { read write execute };
```

Comment:
The patch won't compile as is as tmpfs_t isn't defined. "type tmpfs_t" needs to be added to the list of requirements at the start of the file.

Reply:
Done. Thanks for pointing this out ?